src/scripts/define.test.js

```diff
@@ -228,8 +227,8 @@ describe("glossary", () => {
 
       const fetcher = cache.mock.calls[0].pop();
 
-      axios.get.mockResolvedValue({
-        data: `
+      fetch.mockResolvedValue({
+        text: async () => `
```

Comment:
ok this looked weird but i looked at the whole file in context and figured it out

Reply:
I'll add a comment that it needs to be formatted like yaml so maybe it looks weird to the next person. Would that help?

Reply:
i don't think it's necessary, it was easy enough to see when i opened the file!